packages/colorpicker/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@zendeskgarden/react-colorpicker",
+  "description": "Components related to colorpicker in the Garden Design System",
+  "license": "Apache-2.0",
+  "author": "Zendesk Garden <garden@zendesk.com>",
+  "homepage": "https://garden.zendesk.com/",
+  "repository": "https://github.com/zendeskgarden/react-components",
+  "bugs": {
+    "url": "https://github.com/zendeskgarden/react-components/issues"
+  },
+  "version": "8.29.3",
+  "main": "dist/index.cjs.js",
+  "module": "dist/index.esm.js",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "../../utils/scripts/build.sh",
+    "build:demo": "../../utils/scripts/build-demo.sh",
```

Comment:
This isn't needed anymore with the move to Storybook. Was this included in the package template process?

Reply:
Please retrofit the `.template` as we find these bits.

Reply:
Updated `.template` 👍🏽